Scripts/Core/EditorVR.Actions.cs

```diff
@@ -0,0 +1,93 @@
+#if !UNITY_EDITORVR
+#pragma warning disable 67, 414, 649
```

Comment:
Rather than disabling the warnings, why not wrap all of the class variables in #if UNITY_EDITORVR? I just did rough pass to check for issues and found none. The one caveat is also wrapping usages of EditorVR.kDefaultHideFlags in #if UNITY_EDITORVR. I'll commit it locally in case we want to go this route.